packages/client3/components/design-system/input/input.tsx

```diff
@@ -163,14 +150,8 @@ export const Input = forwardRef<HTMLInputElement, InputProps>(function Input(
       {helperText || _errorMessage ? (
         <HelperText
           className={clsx(
-            isError
-              ? "text-clay-500"
-              : colorScheme === "light"
-              ? "text-sand-500"
-              : colorScheme === "dark"
-              ? "text-sand-300"
-              : null,
-            "mt-1 text-sm leading-none"
+            isError ? "" : null,
+            "text-md mt-1 leading-none text-state-error"
```

Comment:
Just to make sure is this style (`text-state-error`) applied for helpertext as well

Reply:
Yeap, confirmed that the - `text-state-error` is applied to the `HelperText`

Reply:
Oops, my bad. I wasn't clear.  My query was whether the  `text-state-error` needs to be applied to `HelperText`?

Reply:
Whoops my bad. Pushed the fix: 


>         <HelperText
>           className={clsx(
>             isError ? "text-state-error" : null,
>             "mt-1 text-sm leading-none"
>           )}
>         >
>           {_errorMessage ? _errorMessage : helperText}
>         </HelperText>